packages/firebase_crashlytics/firebase_crashlytics_platform_interface/pubspec.yaml

```
@@ -8,16 +8,16 @@ environment:
   flutter: ">=1.12.13+hotfix.5 <2.0.0"
 
 dependencies:
+  collection: ^1.14.3
+  firebase_core: ^0.7.0-nullsafety.0
   flutter:
     sdk: flutter
   meta: ^1.0.5
-  collection: ^1.14.3
-  firebase_core: ^0.7.0
   plugin_platform_interface: ^1.0.2
```

This needs bumping to use a nullsafety version

```suggestion
  plugin_platform_interface: ^1.1.0-nullsafety.1
```